enhancements/authentication/allow-users-to-manage-their-own-tokens.md

```diff
@@ -112,17 +107,32 @@ verbs:
 
 Since the new API allows to remove user-owned access tokens, it should no longer
 be necessary to keep the `system:oauth-token-deleter` clusterrole bound to the
-`system:authenticated` group. On the contrary, this opens a small security hole
-where a user might be able to delete a token of a different user (although, due to
-the default token name length, it is quite improbable). The clusterrolebinding
-`system:oauth-token-deleters` will therefore get deprecated as described in
+`system:authenticated` group. On the contrary, the currently existing clusterrole
+binding means a remaining, small security hole where a user can delete tokens
+that he only knows the sha256 hash of (but it cannot be used to login). To address
+this security hole, the clusterrolebinding `system:oauth-token-deleters` will
+therefore get deprecated as described in
 [Graduation Criteria - Removing a Deprecated Feature](#graduation-criteria---removing-a-deprecated-feature).
 
+#### Default table view in CLI
+
+By default, the following fields will be displayed when listing tokens in the
+command line:
+
+- metadata.name (`NAME`)
+- userName (`USERNAME`)
+- clientName (`CLIENT NAME`)
+- metadata.creationTimestamp (`CREATED`)
```

Comment:
isn't this a default field shown at the very end?

Reply:
we're defining our own table printer, I don't think it would be printed if we don't specifically add it

Reply:
Maybe that is different in CRDs. I remotely remember something like that.